### ISSUE TITLE
Unknown error! 80071a90 installing npcap r-7

### ISSUE THREAD

**Issue by XXXCOBO** [nmap/nmap#1408]
I am confused i have tried everything recommended on here but still cant figure out how to make it work

**Comment by rugabunda** [comment]
me too, had two different error codes but it fails 

Unknown error! 8e5e0408
Npcap LWF driver has failed to be installed.
Failed to create the npcap service for Win7, Win8 and Win10

Unknown error! 8000ffff
Npcap LWF driver has failed to be installed.
Failed to create the npcap service for Win7, Win8 and Win10

**Comment by rugabunda** [comment]
Looks like I have figured out the culprit in my circumstance. This results when the npcap service & certain npcap drivers fails to install, the npcap loopback interface then fails to install, resulting in problems such as this. My driver finally succeeded installing after temporarily disabling group policy `Admin Templates\System\Device Installation\Device Installation Restrictions\Prevent Installation of devices using drivers that match these device setup classes\ (show) {D48179BE-EC20-11D1-B6B8-00C04FA372A7}`

This class of driver was specifically blocked from installing, which is disabled as part of windows in built USB DMA attack protection for bitlocker, via Thunderbolt & SBP2 devices: https://support.microsoft.com/en-us/help/2516445/blocking-the-sbp-2-driver-and-thunderbolt-controllers-to-reduce-1394-d

    IEEE 1394 Devices That Support the SBP2 Protocol Class = SBP2 ClassGuid = {d48179be-ec20-11d1-b6b8-00c04fa372a7}

Strangely, even after re-enabling this policy, the installation problem doesn't seem to be persisting; Why? I don't know. This was the first time in months I was able to install Npcap. The only other policy and or setting I had changed was setting group policy enabled  `"Boot-Start Driver Initialization Policy" to "Good only".`
